python/cli/prelude_cli/views/build.py

```diff
@@ -5,7 +5,7 @@
 import importlib.resources as pkg_resources
 
 from rich import print_json
-from datetime import datetime
+from datetime import datetime, timezone
```

Comment:
can you bump this down a line (we order our imports by length. we know its weird ha)

Reply:
like insert \n between lines 7 and 8?

Reply:
```
from rich ...
from pathlib import Path, PurePath
from datetime import datetime, timezone
```

It's a little weird, but until we fix all the imports everywhere, i'd like to stay consistent

Reply:
ah gotcha. done and done :)